storybook/src/components/Hint/Hint.docs.mdx

```diff
@@ -0,0 +1,21 @@
+{/* @license CC0-1.0 */}
```

Comment:
Do we want to show Hint in Storybook? It's sort of an internal component, no? We could even exclude it from our React export.

Reply:
I was considering that, but decided that it could be handy to expose it. Because: why not? What's against giving developers a feature like this?

Reply:
It's mostly that we do not want them to use it anywhere else at the moment, right? If we show it, it's easier for users to use it incorrectly.

Reply:
I've removed the story and docs 😢